static/responses/json/3/action/package_search?sort=id+asc&start=0&rows=100

```diff
@@ -132,8 +132,8 @@
         "organization": {
           "description": "This is an example organization publishing only example datasets.",
           "created": "2012-06-27T14:48:40.244951",
-          "title": "Example Organization #1",
-          "name": "example-organization-number-one",
+          "title": "Test Publisher 1",
+          "name": "test-publisher-1",
```

Comment:
The reason I called this "Example" rather than "Test" is in case we end up in a situation where this _does_ end up visible on production (maybe it has to for some reason), "Example" looks more obvious and intentional than "Test".

Reply:
Hopefully, it won't reach production Find, I can change the publisher title and name to example, will have to update the ckan functional test as well to handle the change.